### PR TITLE
Overlaid hists

### PR DESCRIPTION
**Pull request by rdoshi99**
Added the overlaid histograms function, and called in the makeplots function as well which should generate the components of it. Also made sure to save the state of the entire exposures/calibrations table initially to ensure that the overlaid_histogram plots can access the overall variable data to compare the distributions.

@ana-lyons I tried adding it to the template like you suggested by putting it inside a similar div container, but was unsuccessful in seeing it actually appear on the generated nightly page.

Also changed the colors of the nightly summary plots to match the corresponding variables on the summary page, and updated the arguments of some functions to ensure that height/width are changeable from the layout and removed an unnecessary argument to the get_timeseries function.

### CODE REVIEW

**Comment by ana-lyons** [comment]
i think that you need to add the components to the jinja template at the end of the makeplots function.

File: py/surveyqa/nightly.py [hunk 1]
```diff
@@ -367,7 +403,11 @@ def makeplots(night, exposures, tiles, outdir):
                     
                     <div class="flex-container">
                         <div>{{ timeseries_script }} {{ timeseries_div }}</div>
-                        <div> NIGHT VS. SUMMARY HISTOGRAMS HERE </div>
+                        <div>{{ overlaidhists_script }} {{ overlaidhists_div }}</div>
+                    </div> 
+
+                    <div class="flex-container">
```
Comment (ana-lyons) [inline py/surveyqa/nightly.py]: also i think you might have duplicate histograms here?

File: py/surveyqa/nightly.py [hunk 2]
```diff
@@ -367,7 +403,11 @@ def makeplots(night, exposures, tiles, outdir):
...
+                        <div>{{ overlaidhists_script }} {{ overlaidhists_div }}</div>
```
Comment (ana-lyons) [inline py/surveyqa/nightly.py]: also I think this is a duplicate histogram?

**Comment by sbailey** [comment]
Good catch, @ana-lyons .  I added the overlaidhists_script/div to the render call:
```
html = jinja2.Template(template).render(
    skypathplot_script=skypathplot_script, skypathplot_div=skypathplot_div,
    exptypecounts_script=exptypecounts_script, exptypecounts_div=exptypecounts_div,
    timeseries_script=timeseries_script, timeseries_div=timeseries_div,
    overlaidhists_script=overlaidhists_script, overlaidhists_div=overlaidhists_div,
    table_script=table_script, table_div=table_div,
    )
```
and now the plots appear.  I made a few other cosmetic changes:
* reduced height of histograms to match height of timeseries plots
* make histogram of this night darker than histogram of overall distribution
* removed title of histogram which was redundant with x-axis label

Ready to merge.

**Pull request merged**